packages/lit-html/src/lit-html.ts

```diff
@@ -491,7 +494,7 @@ const getTemplateHtml = (
           s.slice(0, attrNameEndIndex) +
             boundAttributeSuffix +
             s.slice(attrNameEndIndex)) + marker
-        : s + marker + (attrNameEndIndex === -2 ? `:${i}` : '');
```

Comment:
I removed the `:` here to economize on a couple bytes since I knew I was adding a bit... Don't think it's required.